go/worker/storage/statesync/checkpoint_sync.go

```diff
@@ -21,8 +21,15 @@ import (
 const (
 	// cpListsTimeout is the timeout for fetching checkpoints from all nodes.
 	cpListsTimeout = 30 * time.Second
-	// cpRestoreTimeout is the timeout for restoring a checkpoint chunk from the remote peer.
-	cpRestoreTimeout = 60 * time.Second
+
+	// cpRestoreChunkTimeout is the timeout for restoring a checkpoint chunk from the remote peer.
+	cpRestoreChunkTimeout = 60 * time.Second
+
+	// cpRestoreTimeout is the timeout for restoring the whole checkpoint from the remote peers.
+	//
+	// As of now it takes ~10-30 min to restore the state from the checkpoint, however the timeout
+	// should be significantly higher to account for the growing state.
+	cpRestoreTimeout = 12 * time.Hour
```

Comment:
Is this timeout even useful? It’s so large, who would actually wait 12 hours just to see a program return an error?